shipyard-controller/handler/eventdispatcher.go

```diff
@@ -98,36 +99,6 @@ func (e *EventDispatcher) OnSequenceTimeout(event models.Event) {
 	e.cleanupQueueOfSequence(models.EventScope{KeptnContext: event.Shkeptncontext})
 }
 
-func (e *EventDispatcher) OnSequencePaused(pause models.EventScope) {
```

Comment:
since the state of a sequence execution is handled centrally within the new data model, there is no need for multiple components updating multiple collections to pause/resume a sequence anymore